xwiki-commons-core/xwiki-commons-job/src/main/java/org/xwiki/job/AbstractRequest.java

```diff
@@ -246,4 +247,13 @@ public void setContext(Map<String, Serializable> context)
     {
         setProperty(PROPERTY_CONTEXT, context);
     }
+
+    /**
+     * @return the map of properties in an {@link UnmodifiableMap}.
+     * @since 10.11
+     */
+    public Map<String, Object> getProperties()
+    {
+        return UnmodifiableMap.decorate(properties);
```

Comment:
Not sure what is the benefit compared to `Collections.unmodifiableMap`.

Reply:
yeah sorry I'm a bit like jenkins today, in a bad shape. :sneezing_face: